models/meshery_remote_provider.go

```diff
@@ -463,3 +463,25 @@ func (l *MesheryRemoteProvider) UpdateToken(w http.ResponseWriter, r *http.Reque
 		})
 	}
 }
+
+// ExtractToken - Returns the auth token and the provider type
+func (l *MesheryRemoteProvider) ExtractToken(w http.ResponseWriter, r *http.Request) {
+	l.TokenStoreMut.Lock()
+	defer l.TokenStoreMut.Unlock()
+
+	tokenString, _ := l.GetToken(r)
```

Comment:
Please don't ignore the error instead, you can log in the next line maybe... (Error check might fail...)